{"name": "Eliminar registros del carrito", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: An element could not be located on the page using the given search parameters.\nStacktrace:\nNoSuchElementError: An element could not be located on the page using the given search parameters.\n    at AndroidUiautomator2Driver.findElOrEls (/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-android-driver/lib/commands/find.js:75:11)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at AndroidUiautomator2Driver.findElOrElsWithProcessing (/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-base-driver/lib/basedriver/commands/find.js:33:12)\n    at AndroidUiautomator2Driver.findElement (/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-base-driver/lib/basedriver/commands/find.js:53:10)\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/log_in_steps.py\", line 12, in step_impl\n    log_in_screen.fill_text(*log_in_screen.txt_username, text=\"standard_user\")\n  File \"/Users/leticiaquezadacalzada/Downloads/Telegram Desktop/baz_behave/Behave/utils/common_actions.py\", line 23, in fill_text\n    text_field = self.find_element(*locator)\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/leticiaquezadacalzada/Downloads/Telegram Desktop/baz_behave/Behave/utils/common_actions.py\", line 12, in find_element\n    return self.driver.find_element(*locator)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/appium/webdriver/webdriver.py\", line 387, in find_element\n    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/appium/webdriver/errorhandler.py\", line 30, in check_response\n    raise wde\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/appium/webdriver/errorhandler.py\", line 26, in check_response\n    super().check_response(response)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Tengo la aplicaci√≤n", "status": "passed", "start": 1688363855629, "stop": 1688363855629}, {"name": "When Ingreso usuario", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: An element could not be located on the page using the given search parameters.\nStacktrace:\nNoSuchElementError: An element could not be located on the page using the given search parameters.\n    at AndroidUiautomator2Driver.findElOrEls (/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-android-driver/lib/commands/find.js:75:11)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at AndroidUiautomator2Driver.findElOrElsWithProcessing (/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-base-driver/lib/basedriver/commands/find.js:33:12)\n    at AndroidUiautomator2Driver.findElement (/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-base-driver/lib/basedriver/commands/find.js:53:10)\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/log_in_steps.py\", line 12, in step_impl\n    log_in_screen.fill_text(*log_in_screen.txt_username, text=\"standard_user\")\n  File \"/Users/leticiaquezadacalzada/Downloads/Telegram Desktop/baz_behave/Behave/utils/common_actions.py\", line 23, in fill_text\n    text_field = self.find_element(*locator)\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/leticiaquezadacalzada/Downloads/Telegram Desktop/baz_behave/Behave/utils/common_actions.py\", line 12, in find_element\n    return self.driver.find_element(*locator)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/appium/webdriver/webdriver.py\", line 387, in find_element\n    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/appium/webdriver/errorhandler.py\", line 30, in check_response\n    raise wde\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/appium/webdriver/errorhandler.py\", line 26, in check_response\n    super().check_response(response)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1688363855630, "stop": 1688363860879}, {"name": "And Ingreso password", "status": "skipped", "start": 1688363861293, "stop": 1688363861293}, {"name": "And Doy click en Ingresar", "status": "skipped", "start": 1688363861293, "stop": 1688363861293}, {"name": "Given Doy click en el carrito", "status": "skipped", "start": 1688363861293, "stop": 1688363861293}, {"name": "When Verifico que el carrito no tenga productos y en caso de tenerlo lo elimino", "status": "skipped", "start": 1688363861293, "stop": 1688363861293}, {"name": "Then Regreso a la pantalla de productos", "status": "skipped", "start": 1688363861293, "stop": 1688363861293}], "start": 1688363843797, "stop": 1688363861293, "uuid": "ddc5a513-a221-44f5-9f05-077157ad5d7d", "historyId": "8c21f5d52e5ece378f9253263f5fd908", "fullName": "Buy Products: Eliminar registros del carrito", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "buy"}, {"name": "tag", "value": "regression"}, {"name": "feature", "value": "Buy Products"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}